### PR TITLE
feat(core): getAllResources method

### PR DESCRIPTION
**Pull request by sarkapalkovicova**
Added method for obtaining all resources from the database.

### CODE REVIEW

File: perun-core/src/main/java/cz/metacentrum/perun/core/implApi/ResourcesManagerImplApi.java [hunk 1]
```diff
@@ -25,6 +25,7 @@
 import cz.metacentrum.perun.core.api.exceptions.GroupAlreadyRemovedFromResourceException;
 import cz.metacentrum.perun.core.api.exceptions.GroupNotDefinedOnResourceException;
 import cz.metacentrum.perun.core.api.exceptions.InternalErrorException;
+import cz.metacentrum.perun.core.api.exceptions.PrivilegeException;
```
Comment (Johaney-s) [inline perun-core/src/main/java/cz/metacentrum/perun/core/implApi/ResourcesManagerImplApi.java:28]: unused exception

File: perun-core/src/test/java/cz/metacentrum/perun/core/entry/ResourcesManagerEntryIntegrationTest.java [hunk 2]
```diff
@@ -95,6 +95,21 @@ public void setUp() {
...
+		vo = setUpVo();
+		facility = setUpFacility();
+		resource = setUpResource();
+		Resource resource2 = setUpResource2();
+
+
+		List<Resource> resources = resourcesManager.getAllResources(sess);
+
+		assertThat(resources).contains(resource, resource2);
+		assertEquals(resources.size(), 2);
```
Comment (Johaney-s) [inline perun-core/src/test/java/cz/metacentrum/perun/core/entry/ResourcesManagerEntryIntegrationTest.java:111]:
```suggestion
		assertEquals(2, resources.size());
```
just a nitpick, the expected value goes first so assertion error has proper message if raised

File: perun-core/src/main/java/cz/metacentrum/perun/core/entry/ResourcesManagerEntry.java [hunk 3]
```diff
@@ -63,6 +63,8 @@
 import java.util.List;
 import java.util.stream.Collectors;
 
+import static java.util.stream.Collectors.toList;
```
Comment (Johaney-s) [inline perun-core/src/main/java/cz/metacentrum/perun/core/entry/ResourcesManagerEntry.java:66]: unused import

File: perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/ResourcesManagerBlImpl.java [hunk 4]
```diff
@@ -55,6 +55,7 @@
 import cz.metacentrum.perun.core.api.exceptions.GroupResourceStatusException;
 import cz.metacentrum.perun.core.api.exceptions.InternalErrorException;
 import cz.metacentrum.perun.core.api.exceptions.MemberResourceMismatchException;
+import cz.metacentrum.perun.core.api.exceptions.PrivilegeException;
```
Comment (Johaney-s) [inline perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/ResourcesManagerBlImpl.java:58]: unused imports

**Review by Johaney-s — APPROVED** [review]

File: perun-core/src/main/java/cz/metacentrum/perun/core/bl/ResourcesManagerBl.java [hunk 5]
```diff
@@ -33,6 +33,7 @@
 import cz.metacentrum.perun.core.api.exceptions.GroupResourceMismatchException;
 import cz.metacentrum.perun.core.api.exceptions.GroupResourceStatusException;
 import cz.metacentrum.perun.core.api.exceptions.InternalErrorException;
+import cz.metacentrum.perun.core.api.exceptions.PrivilegeException;
```
Comment (xflord) [inline perun-core/src/main/java/cz/metacentrum/perun/core/bl/ResourcesManagerBl.java:36]: unused import

File: perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/ResourcesManagerBlImpl.java [hunk 6]
```diff
@@ -87,6 +87,7 @@
 import java.util.Optional;
 import java.util.stream.Collectors;
 
+import static java.util.stream.Collectors.toList;
```
Comment (xflord) [inline perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/ResourcesManagerBlImpl.java:90]: unused import

**Review by xflord — APPROVED** [review]

**Pull request merged**